#!/usr/bin/env python
PACKAGE = "emergency_stop_recover_pa"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pointcloudTopic"   , str_t   , 0, "Topic Name of input pointcloud"         , "/input_cloud")
gen.add("baseLinkFrame"     , str_t   , 0, "Frame ID of the base link"              , "/base_link")
gen.add("velocityTopic"     , str_t   , 0, "Topic name of the subscribed velocity"  , "/recover/cmd_vel")
gen.add("inlierThresh"      , int_t   , 0, "Threshold for inlier counter"           , 5)
gen.add("recover_step_vel"  , double_t, 0, "Step Increment for recover velocity"    , 0.1)
gen.add("recover_factor_vel", int_t   , 0, "Multiplying factor for recover velocity", 1)

exit(gen.generate(PACKAGE, "emergency_stop_recover_pa", "region"))
